modules/editor-events/assets/js/editor/module.js

```diff
@@ -0,0 +1,48 @@
+import EventsStorage from './events-storage';
+
+export default class extends elementorModules.Module {
+	types = {
+		click: 'click',
+	};
+
+	events = {
+		site_settings: 'site_settings',
+	};
+
+	onInit() {
+		if ( ! elementor.config.editor_events?.can_send_events ) {
+			return;
+		}
+
+		window.addEventListener( 'unload', this.sendEvents() );
```

Comment:
I think that this event is deprecated, [Here](https://developer.mozilla.org/en-US/docs/Web/API/Window/unload_event#usage_notes)

Reply:
updated to `beforeunload`, it's also ok I guess